tests/module_1/test_meteo_api.py

```diff
@@ -1,7 +1,42 @@
-""" This is a dummy example to show how to import code from src/ for testing"""
+import pandas as pd
 
-from src.module_1.module_1_meteo_api import main
+from src.module_1.module_1_meteo_api import process_data, VARIABLES
 
 
-def test_main():
-    raise NotImplementedError
\ No newline at end of file
+def test_process_data():
```

Comment:
Add tests for other functions, specially for `clean_data`

Reply:
As I deleted clean_dat for not being used, I think it will be a good idea to mock responses using call_api/get_data_meteo_api funcs

Reply:
Tests implemented using MockResponses provided in the example code. I need to fully digest the understanding and uses of mocking functions for future implementations. I modified the call_api functions and now all tests passed